catalog/app/containers/Admin/selectors.js

```diff
@@ -0,0 +1,36 @@
+import { Map } from 'immutable';
+import { createSelector } from 'reselect';
+
+/**
+ * Direct selector to the admin state domain
+ */
+export const selectAdminDomain = (state) => state.get('admin', Map({}));
+
+/**
+ * Other specific selectors
+ */
+
```

Comment:
these comment blocks should be on one line and unused comment blocks deleted

Reply:
ok. they just were there and i didnt touch them